src/main/java/seedu/address/model/task/Task.java

```diff
@@ -26,7 +26,7 @@
     private TaskType taskType;
     private RecurringType recurringType;
     
-    private List<TaskComponent> recurringDates;
+    public List<TaskComponent> recurringDates;
```

Comment:
Not sure why the private List is changed to public.
Do we not have a getComponentList() method?